### PR TITLE
Upgrade to React 16.8.6

### PR DESCRIPTION
**Pull request by vutoan1245**
Upgrade to React 16.8.6

Resolve issue #879

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,5 +71,9 @@
 	"engines": {
 		"yarn": "^1.0",
 		"node": "^10.4.1"
+	},
+	"dependencies": {
```
Comment (iturgeon) [inline package.json]: I don't think these should be added at the top level project.json, will it work without it?

File: packages/util/obojobo-lib-utils/package.json [hunk 2]
```diff
@@ -47,8 +47,8 @@
 		"jest": "24.1.0",
 		"lint-staged": "^6.1.1",
 		"prettier": "1.11.1",
-		"react-dom": "16.7.0",
-		"react-test-renderer": "16.7.0",
+		"react-dom": "16.8.6",
```
Comment (iturgeon) [inline packages/util/obojobo-lib-utils/package.json]: I believe we shouldn't have react-dom here under devDeps since its already under regular deps.

**Comment by iturgeon** [comment]
Oh, wanted to add, this should have been pushed to your fork.  I changed some permission settings given some new features Github has, we'll see if that'll do what we want.

**Comment by iturgeon** [comment]
Forgot to mention, I quickly tested the viewer and editor, the jump to a newer React seems to work just fine.

**Comment by vutoan1245** [comment]
@iturgeon Sorry I was testing Github Desktop, not sure why it pushes directly to the repo.

**Comment by vutoan1245** [comment]
I will close this PR and make a new one.

**Pull request closed without merging**